### DISCUSSION TITLE
Request to join team

### DISCUSSION THREAD

**Discussion by krishnaspeace** [ROM-builders/temporary#22203] [General]
Tree links:
https://github.com/krishnaspeace/android_device_xiaomi_ysl
https://github.com/krishnaspeace/vendor_xiaomi_ysl
https://github.com/krishnaspeace/android_device_xiaomi_msm8953-common
https://github.com/krishnaspeace/android_kernel_xiaomi_msm8953

My successful builds:
https://cirrus-ci.com/build/5889296142434304
https://cirrus-ci.com/build/4624937730703360
https://cirrus-ci.com/build/6520555906531328

Telegram username: https://t.me/krishanspeace
GitHub username: krishnaspeace

my previous github account - aaravmujawdia was deleted i request you to please accept these new account request

**Comment by Apon77** [comment]
Please check #learn instruction from telegram group.
